config.ini

```
@@ -20,6 +20,7 @@
 debug           = 0             # 0 = NO, 1 = YES
 use_pvoutput    = 1             # 0 = NO, 1 = YES       to export data to http://pvoutput.org
 use_rrdtool     = 0             # 0 = NO, 1 = YES       to export data to rrdtool for graphing
+use_mqtt        = 1             # 0 = NO, 1 = YES       to export data to mqtt

```

Because this isn't supported by me, let's leave this off by default.

```suggestion
use_mqtt        = 0             # 0 = NO, 1 = YES       to export data to mqtt
```